src/index.ts

```diff
@@ -81,7 +96,7 @@ export default function create<
     return stateSlice
   }
 
-  let state = createState(setState as SetState, getState as GetState)
+  let state = createState(setState as TSetState, getState as TGetState)
   const api = { destroy, getState, setState, subscribe }
 
   return [useStore, api] as [typeof useStore, typeof api]
```

Comment:
`as [typeof useStore, typeof api]` is not needed because you correctly defined the return type in the function signature.
```diff
- const api = { destroy, getState, setState, subscribe }
-
- return [useStore, api] as [typeof useStore, typeof api]
+ return [useStore, { destroy, getState, setState, subscribe }]
```